foursight_core/deploy.py

```diff
@@ -173,6 +178,9 @@ def build_config_and_package(cls, args, identity=None, stack_name=None, trial_cr
             environment variables, a list of security group ids, and a list of subnet ids. Finally, packages as a
             Cloudformation template."""
 
+        # TODO: This passes rds_name through, but build_config is no longer using it. Is that correct?
+        #       Should this be just marking that arg ignored or just not receiving such an arg?
+        #       Something is weird in that data flow chain. -kmp 19-Aug-2022
```

Comment:
Oh, yes, sorry about that; RDS_NAME will now be in the GAC (with newer, in PR, 4dn-cloud-infra); I will add that the the CHANGELOG.rst. I should remove the rds_name arg now; will take care of that.